lib/client-assets.php

```diff
@@ -566,7 +566,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Initialize the editor.
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() { wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost ); } );' );
+	$editor_settings = array(
+		'wideImages' => get_theme_support( 'wide-images' ),
```

Comment:
Good questions, I'm happy to use whatever we settle on. So you're proposing something like:

```php
add_theme_support( 'gutenberg-theme-support', [ 'wide-images' => true ] );
```
I kind of like the one liner if you want to enable a feature (I mean the current approach) but no strong opinion here. cc @jasmussen